### ISSUE TITLE
sys.meta_path is None, Python is likely shutting down

### ISSUE THREAD

**Issue by alecxe** [SeleniumHQ/selenium#3330]
Reproduced using:

Python 3.6.0 (latest at the moment)
Selenium 3.0.2 (latest at the moment)
ChromeDriver 2.27 (latest at the moment) 
Mac OS 

The code I'm executing:

```python
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Chrome(executable_path="/usr/local/bin/chromedriver")
driver.get('http://videohost.site/play/A11QStEaNdVZfvV/')

video = WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.TAG_NAME, "video")))
print(video.get_attribute("src"))

driver.close()
```

The error I'm getting:

```
Exception ignored in: <bound method Service.__del__ of <selenium.webdriver.chrome.service.Service object at 0x101b509b0>>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/selenium/webdriver/common/service.py", line 173, in __del__
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/selenium/webdriver/common/service.py", line 149, in stop
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/selenium/webdriver/common/service.py", line 120, in send_remote_shutdown_command
ImportError: sys.meta_path is None, Python is likely shutting down
```

Why is this happening?

Note: I can workaround it using `driver.quit()` instead of `driver.close()`.

Thanks!

Cross-linking the SO post: http://stackoverflow.com/q/41480148/771848

**Comment by cgoldberg** [comment]
#3263 PR is related

**Comment by lmtierney** [comment]
Let's assume this was fixed with #3263, if not please raise a new issue.
